docs/source/releases/latest/852-remove-unused-sphinx-extensions.yaml

```diff
@@ -0,0 +1,18 @@
+removal:
+- title: 'Remove Unused Sphinx Extensions'
+  description: 'Removed unused Sphinx extensions to simplify build process'
```

Comment:
How did you determine which extensions were unused? Did you confirm a fresh install without these extensions worked for all plug-in packages, with all builds ?